website/cue/reference/components/sinks.cue

```diff
@@ -627,10 +627,13 @@ components: sinks: [Name=string]: {
 	}
 
 	telemetry: metrics: {
-		events_in_total:                      components.sources.internal_metrics.output.metrics.events_in_total
-		events_out_total:                     components.sources.internal_metrics.output.metrics.events_out_total
 		component_received_events_total:      components.sources.internal_metrics.output.metrics.component_received_events_total
 		component_received_event_bytes_total: components.sources.internal_metrics.output.metrics.component_received_event_bytes_total
+		component_sent_bytes_total:           components.sources.internal_metrics.output.metrics.component_sent_bytes_total
+		component_sent_events_total:          components.sources.internal_metrics.output.metrics.component_sent_events_total
+		component_sent_event_bytes_total:     components.sources.internal_metrics.output.metrics.component_sent_event_bytes_total
```

Comment:
Not all sinks support this yet, right? Should we avoid adding it here until they do? Instead we should just add it to the sinks that use `HttpSink`?

Reply:
Correct, not all sinks emit them yet. However, AFAICT no sink actually emits `events_out_total` before this, and it was listed there, so I followed the pattern.

Reply:
🤔 I think `events_out_total` shouldn't be listed here either if it isn't emitted by all components. I realize it would be tedious to go through and add this metric to only the components that are reporting it though; especially if we eventually aim to have all reporting and move it back to here.

Reply:
Done

Reply:
Thank you! This will save users some confusion if we release before we are able to roll this out to all components.